src/cts/src/CtsOptions.h

```diff
@@ -300,6 +307,9 @@ class CtsOptions
   float sinkBufferMaxCapDerateDefault_ = 0.01;
   float sinkBufferMaxCapDerate_ = sinkBufferMaxCapDerateDefault_;
   bool dummyLoad_ = false;
+  float delayBufferDerateSet_ = false;
+  float delayBufferDerateDefault_ = 1.0;  // no derate
+  float delayBufferDerate_ = delayBufferDerateDefault_;
```

Comment:
delayBufferDerateDefault_ is const.   I would find
```
  float delayBufferDerate_ = 1.0;  // no derate;
```
just as clear and a bit simpler.

Reply:
Done